### PR TITLE
Building the bootloader commands update

### PR DESCRIPTION
**Pull request by RicardoM17**
It can be quite obvious for certain people but if there is a list of commands it should probably include these as well.

### CODE REVIEW

**Review by mhkabir — APPROVED** [review]

File: en/software_update/stm32_bootloader.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The code for the PX4 bootloader is available from the Github [Bootloader](https:
 ```bash
 git clone https://github.com/PX4/Bootloader.git
 cd Bootloader
+git submodule init
```
Comment (hamishwillee) [inline en/software_update/stm32_bootloader.md:19]: Why not simpler invocation?

```bash
git clone https://github.com/PX4/Bootloader.git
cd Bootloader
git submodule update --init --recursive
```

Comment (dagar) [reply en/software_update/stm32_bootloader.md:19]: You can be even lazier.
`git clone --recursive https://github.com/PX4/Bootloader.git`

Comment (hamishwillee) [reply en/software_update/stm32_bootloader.md:19]: That is very lazy :-). I tend to have the `--recursive` flag at the end.

Comment (hamishwillee) [reply en/software_update/stm32_bootloader.md:19]: I'm going to leave as is. Merging

**Pull request merged**